src/components/library-item/library-item.css

```diff
@@ -48,11 +49,30 @@
     width: 100%;
 }
 
+.library-item-inset-image-container {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-top: -2.5rem;
+    margin-left: 1rem;
+    align-self: flex-start;
+    background-color: rgb(35, 188, 141);
```

Comment:
I think this color is already defined as `$pen-primary`

Reply:
Cool thanks!